lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -362,4 +362,20 @@ sub setup_vm_simple_dns_with_ip {
     record_info("Simple DNS setup in /etc/hosts for $_ip $_vm is successful!", script_output("cat /etc/hosts"));
 }
 
+sub get_guest_status {
+    my ($guest, %args) = @_;
+    my $timeout = $args{timeout} // "180";
```

Comment:
I do no think this parameter  my $timeout is necessary.

Reply:
yes, good point. keep this $timeout just only to ensure future expansion for this function.